### PR TITLE
Fix install routine

### PR DESCRIPTION
**Pull request by wazolab**

### CODE REVIEW

File: vidos.config.sample.ts
Comment (wazolab) [inline vidos.config.sample.ts:1]: I'm just wondering if we should update this file with working env variables ?
@frodrigo : Do you how something to suggests ?

Comment (wazolab) [reply vidos.config.sample.ts:1]: When a new developer will clone this repo I will have to copy this sample to `vidos.config.ts`.  But it's a "sample" so it shouldn't contain any "working" values right ?

File: vidos.config.sample.ts [hunk 1]
```diff
@@ -5,10 +5,9 @@ const configDefault = {
   API_SEARCH_ADDR: 'https://api-adresse.data.gouv.fr/search',
   API_EXPORT: 'https://print-dev.teritorio.xyz/v0.1',
   API_QR_SHORTENER: 'https://qr-shortener-dev.teritorio.xyz',
-  BICYCLE_STYLE_URL:
-    'https://vecto.teritorio.xyz/styles/teritorio-tourism-bicycle-0.9/style.json',
   IMAGE_PROXY: null,
-  MAPILLARY_ACCESS_TOKEN: null,
+  MAPILLARY_ACCESS_TOKEN: 'MLY|5609419529131330|36d86a5578fecba521c1ffe033bda2fc',
```
Comment (frodrigo) [inline vidos.config.sample.ts:9]: Project code should not contain key values.

File: vidos.config.sample.ts [hunk 2]
```diff
@@ -21,15 +20,12 @@ const config: VidosConfig = {
...
     API_THEME: 'tourism',
-    VECTO_STYLE_URL:
-      'https://merge-proxy-dev.teritorio.xyz/styles/teritorio-tourism-0.9/style.json?key=###',
-    SATELLITE_STYLE_URL:
-      'https://merge-proxy-dev.teritorio.xyz/styles/satellite-hybrid/style.json?key=###',
-    BICYCLE_STYLE_URL:
-      'https://merge-proxy-dev.teritorio.xyz/styles/teritorio-tourism-bicycle-0.9/style.json?key=###',
+    VECTO_STYLE_URL: 'https://merge-proxy.teritorio.xyz/styles/teritorio-tourism-1.0/style.json?key=landesatlantiquesud-1-ood5saeCho6quei6ees3Ut',
+    SATELLITE_STYLE_URL: 'https://merge-proxy.teritorio.xyz/styles/satellite-hybrid/style.json?key=landesatlantiquesud-1-ood5saeCho6quei6ees3Ut',
+    BICYCLE_STYLE_URL: 'https://merge-proxy.teritorio.xyz/styles/teritorio-bicycle-tourism/style.json?key=landesatlantiquesud-1-ood5saeCho6quei6ees3Ut',
```
Comment (frodrigo) [inline vidos.config.sample.ts:28]: Project code should not contain key values.

**Pull request merged**